schema/notifications.hcl

```diff
@@ -0,0 +1,77 @@
+table "notifications" {
+  schema = schema.public
+  column "id" {
+    null    = false
+    type    = uuid
+    default = sql("generate_ulid()")
+  }
+  column "events" {
+    null    = false
+    type    = sql("text[]")
+    comment = "a list of events this notification is for."
+  }
+  column "template" {
+    null = false
+    type = text
+  }
+  column "filter" {
+    null = true
+    type = text
+  }
+  column "properties" {
+    null    = true
+    type    = jsonb
+    comment = "Shoutrrr properties used when sending email to the person receipient."
+  }
```

Comment:
@moshloop I didn't remove this as we still need customization when sending email (like from name and subject)